packages/studio-base/src/components/PlaybackControls/index.tsx

```diff
@@ -230,43 +230,71 @@ export default function PlaybackControls(props: {
             <PlaybackTimeDisplay onSeek={seek} onPause={pause} />
           </Stack>
           <Stack direction="row" alignItems="center" gap={1}>
-            <HoverableIconButton
-              disabled={disableControls}
-              size="small"
-              title="Seek backward"
-              icon={<Previous20Regular />}
-              activeIcon={<Previous20Filled />}
-              onClick={() => {
-                seekBackwardAction();
-              }}
-            />
-            <HoverableIconButton
-              disabled={disableControls}
-              size="small"
-              title={isPlaying ? "Pause" : "Play"}
-              onClick={togglePlayPause}
-              icon={isPlaying ? <Pause20Regular /> : <Play20Regular />}
-              activeIcon={isPlaying ? <Pause20Filled /> : <Play20Filled />}
-            />
-            <HoverableIconButton
-              disabled={disableControls}
-              size="small"
-              title="Seek forward"
-              icon={<Next20Regular />}
-              activeIcon={<Next20Filled />}
-              onClick={() => {
-                seekForwardAction();
-              }}
-            />
+            <Tooltip
+              enterDelay={1200}
+              enterNextDelay={800}
+              title={
+                <>
+                  Seek backward <kbd>←</kbd>
+                </>
+              }
+            >
+              <HoverableIconButton
+                disabled={disableControls}
+                size="small"
+                icon={<Previous20Regular />}
+                activeIcon={<Previous20Filled />}
+                onClick={() => {
```

Comment:
We can get rid of this wrapping function:
```suggestion
                onClick={seekBackwardAction
```